cloudbuild.yaml

```diff
@@ -5,4 +5,7 @@ options:
 steps:
 - name: gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20240718-5ef92b5c36
   entrypoint: make
+  env:
+  - REGISTRY=gcr.io/k8s-staging-networking
+  - IMAGE_NAME=kube-network-policies
   args: ['release']
```

Comment:
question: do we need to get gcloud auth for docker imager here?, just like `gcloud auth configure-docker`

cc @ameukam , thank you!

see https://github.com/kubernetes-sigs/kwok/blob/2033297e7e4bb3068b1d8ef4d240f9decab0663c/cloudbuild.yaml#L23